### ISSUE TITLE
Multiple bugs with copyToken() macro function

### ISSUE THREAD

**Issue by Phergus** [RPTools/maptool#1300]
**Describe the bug**
Executing the copyToken function without the `updates` parameter throws an NPE and fails to make copy.
Executing the copyToken function with all parameters makes the copies but returns a bogus error message.

**To Reproduce**
1. With Hero token on current map execute [r: copyToken("Hero")] in chat.
2. Produces
  `java.lang.NullPointerException error executing expression copyToken("Hero").`

3. Executing 
  ```[h: cloneNo = 0]
[h: updates = "{name: 'Hero Clone - [r: cloneNo = cloneNo + 1]'}"]
[h: copyToken("Hero", 2, "Grasslands", updates)]
```
4. Produces the two tokens but also returns:
  `Argument number 1 to function "copyToken" must be a Token id or name.`

**Expected behavior**
Expect no error messages and in the first case the ID of the new token.  In the second case a JSON array of the two token IDs.

**MapTool Info**
- Version: 1.5.11+
- Install: Any

**Desktop (please complete the following information):**
 - OS: Windows
 - Version: 10

**Comment by aliasmask** [comment]
This is still an issue: https://forums.rptools.net/viewtopic.php?f=20&t=28602&p=275547#p275537

**Comment by Phergus** [comment]
@aliasmask  Right.  Because it's only in 1.6.0 alpha 7.

**Comment by aliasmask** [comment]
Issue/bug is in current version and before, unless you mean the fix is only in 1.6.

**Comment by Phergus** [comment]
>  Right. Because it's **only** in 1.6.0 alpha 7.

**Comment by aliasmask** [comment]
If you look at the link, that is not the case, as I said above. In 1.5.12, 1.5.13 (probably .11 too) if you type copyToken("tokenName") it will give NPE error.

**Comment by Phergus** [comment]
The fix is only in 1.6.0 alpha 7.

**Comment by aliasmask** [comment]
Ah, "it's" was the fix, not the issue. So, I don't expect 1.6.0 to be released anytime soon based on some of the goals for its release. Will the fix be pushed to next 1.5 release?

**Comment by Phergus** [comment]
No plans currently for a next 1.5 release but that can be discussed.

**Comment by Phergus** [comment]
Released in 1.5.14.  Closing.
